infrastructure/swagger/04_apim.yaml

```diff
@@ -12,4 +12,7 @@ x-nhsd-apim:
     security:
       type: apikey
       header: apikey
-      secret: rowan-cpm-test-5
+      secret: rowan-cpm-test-8
```

Comment:
We can't push secrets into github.  We need to figure out how to handle this.

Reply:
Thats the name of the secret and not the secret itself - and its stored within APIMs setup